crates/rpc-types/src/eth/transaction/mod.rs

```diff
@@ -86,6 +87,33 @@ pub struct Transaction {
     pub other: OtherFields,
 }
 
+impl Transaction {
+    /// Converts [Transaction] into [TransactionRequest].
+    ///
+    /// During this conversion data for [TransactionRequest::sidecar] is not populated as it is not
+    /// part of [Transaction].
+    pub fn into_request(self) -> TransactionRequest {
```

Comment:
any reason we have this instead of just the auto-impl'd `Into` we get from `From`?

Reply:
We wanted to have it documented on `Transaction`  

Ref https://github.com/alloy-rs/alloy/pull/338#discussion_r1529052619